### PR TITLE
Mapped some non-realms stuff

### PR DESCRIPTION
**Pull request by liach**
Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

File: mappings/net/minecraft/entity/passive/VillagerEntity.mapping [hunk 1]
```diff
@@ -8,11 +8,12 @@ CLASS net/minecraft/class_1646 net/minecraft/entity/passive/VillagerEntity
...
 	FIELD field_18535 gossipStartTime J
 	FIELD field_18536 experience I
-	FIELD field_18537 lastRestock J
+	FIELD field_18537 lastRestockTime J
 	FIELD field_18538 MEMORY_MODULES Lcom/google/common/collect/ImmutableList;
 	FIELD field_18539 SENSORS Lcom/google/common/collect/ImmutableList;
 	FIELD field_18851 POINTS_OF_INTEREST Ljava/util/Map;
-	FIELD field_19357 lastGossipDecay J
+	FIELD field_19357 lastGossipDecayTime J
+	FIELD field_19427 todayRestock I
```
Comment (Runemoro) [inline mappings/net/minecraft/entity/passive/VillagerEntity.mapping]: What is this?

File: mappings/net/minecraft/util/CsvOutput.mapping [hunk 2]
```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_4456 net/minecraft/util/CsvOutput
+	CLASS class_4457 Header
+		FIELD field_20286 columns Ljava/util/List;
+		METHOD method_21631 startBody (Ljava/io/Writer;)Lnet/minecraft/class_4456;
+		METHOD method_21632 column (Ljava/lang/String;)Lnet/minecraft/class_4456$class_4457;
```
Comment (Runemoro) [inline mappings/net/minecraft/util/CsvOutput.mapping]: This is a method. Shouldn't it be something like `addColumn`?

File: mappings/net/minecraft/util/CsvOutput.mapping [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+	FIELD field_20284 writer Ljava/io/Writer;
+	FIELD field_20285 column I
+	METHOD <init> (Ljava/io/Writer;Ljava/util/List;)V
+		ARG 1 writer
+		ARG 2 headers
+	METHOD method_21627 header ()Lnet/minecraft/class_4456$class_4457;
```
Comment (Runemoro) [inline mappings/net/minecraft/util/CsvOutput.mapping]: This should be `getHeader`.

File: mappings/net/minecraft/entity/passive/VillagerEntity.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/entity/passive/VillagerEntity.mapping]: Look at **RestocksToday** in https://minecraft.gamepedia.com/Villager/ED

File: mappings/net/minecraft/util/CsvOutput.mapping (hunk 3 above)
Comment (liach) [reply mappings/net/minecraft/util/CsvOutput.mapping]: This is like a `builder()` method.

File: mappings/net/minecraft/util/CsvOutput.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/util/CsvOutput.mapping]: I originally named this like a builder. It worthes some debate on whether this should be as is or be named as a builder.

File: mappings/net/minecraft/util/CsvOutput.mapping [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+	METHOD method_21628 print (Ljava/lang/Object;)Ljava/lang/String;
+		ARG 0 object
+	METHOD method_21629 row (Ljava/util/stream/Stream;)V
+		ARG 1 columns
+	METHOD method_21630 row ([Ljava/lang/Object;)V
+		ARG 1 columns
```
Comment (Runemoro) [inline mappings/net/minecraft/util/CsvOutput.mapping]: `printRow`?

File: mappings/net/minecraft/entity/passive/VillagerEntity.mapping (hunk 1 above)
Comment (Runemoro) [reply mappings/net/minecraft/entity/passive/VillagerEntity.mapping]: `restocksToday` is clear that it's the number of restocks today. `todayRestock` doesn't make sense.

File: mappings/net/minecraft/util/CsvOutput.mapping [hunk 5]
```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_4456 net/minecraft/util/CsvOutput
```
Comment (Runemoro) [inline mappings/net/minecraft/util/CsvOutput.mapping]: Maybe the class can be named something like `CsvWriter` (similar to how we have `java.io.Writer`)?

**Review by Runemoro — APPROVED** [review]

**Comment by liach** [comment]
On a side note, this should have no need for an update as this does not touch any constuctors or broken methods. Will add the new long argument stuff and new villager task in as well.

**Review by asiekierka — APPROVED** [review]

**Pull request merged**